dirsrvtests/tests/suites/replication/moddn_off_and_entrydn_test.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+import os
+import time
+from lib389._constants import (
+    DN_CONFIG_LDBM, DEFAULT_SUFFIX, DEFAULT_BENAME,
+    REPLICA_PRECISE_PURGING, REPLICA_PURGE_DELAY, REPLICA_PURGE_INTERVAL)
+from lib389.topologies import topology_m1 as topo
+from lib389.dseldif import DSEldif
+from lib389.tombstone import Tombstones
+from lib389.idm.user import UserAccounts, TEST_USER_PROPERTIES
+
+
```

Comment:
I think we need to add `pytestmark = pytest.mark.tier0` here.

And could you also please add Copyright to the file?